docs/guide/deployer.md

```diff
@@ -27,6 +27,22 @@ You can write deployment tasks synchronously and they'll be executed in the corr
 
 To deploy an ASA you must have `asa.yaml` file in `assets` folder. Head to the [ASA Definition File Spec](https://paper.dropbox.com/doc/Algorand-builder-specs-Vcdp0XNngizChyUWvFXfs#:uid=077585002872354521007982&h2=ASA-Definition-File) to learn more.
 
+#### Deploying ASA with custom parameters
+
+User can also override the fields in `assets/asa.yaml` when deploying ASA. Eg. If user wants to use a multisig address from 3 accounts: `alice, bob, john` as the Asset Reserve. We can create a multisig address in `algob` script first and then pass this address as a custom asaParams.
```

Comment:
are we overwriting fields in .yaml ? because that's the current deployed asa.
Please update `override`

Reply:
I think overwriting means that we overwrite .yaml as well. Override means that we override the input from .yaml file with custom `asaParams`. So not sure if this should be overwrite

Reply:
That's why I mentioned to overwrite

Reply:
You're suggesting to update the implementation to "overwrite" asa.yaml as well. OK.
By your previous comment it seams you suggest to update the readme only.